Delphi.WebMock.RequestStub.pas

```diff
@@ -117,6 +126,14 @@ function TWebMockRequestStub.WithBody(
   Result := Self;
 end;
 
+function TWebMockRequestStub.WithBody(const ADataName, ADataValue: string): TWebMockRequestStub;
+var LValue: string;
+begin
+  LValue := 'form-data; name="%s"' + sLineBreak + sLineBreak + '%s' + sLineBreak;
+  LValue := Format(LValue, [ADataName, ADataValue]);
```

Comment:
I posted this because my implementation required it, with a day to think about it I might implement that method with a trailing enum parameter: `TFormFieldsKind = (ffkMultipart, ffkList)`, and those parameters should then match the defaults used by Delphi.
 
Also it doesn't necessarily require TRegex an IsMatching function would be enough but you haven't implemented that yet. Just trying to keeps things simple. 

As for your preference, go at it. I simply needed this and thought you could use some of it to extend WebMocks.